compiler/rustc_borrowck/src/type_check/constraint_conversion.rs

```diff
@@ -286,7 +287,7 @@ impl<'a, 'tcx> ConstraintConversion<'a, 'tcx> {
                     ocx.infcx.at(&ObligationCause::dummy_with_span(self.span), self.param_env),
                     ty,
                 )
-                .map_err(|_| NoSolution)
+                .map_err(|_: Vec<ScrubbedTraitError<'tcx>>| NoSolution)
```

Comment:
this can use `ocx.deeply_normalize` instead, so we don't have to specify `ScrubbedTraitError`